tools/mypy/defs.bzl

```diff
@@ -0,0 +1,15 @@
+load("@pip_types//:types.bzl", "types")
+load("@rules_mypy//mypy:mypy.bzl", "mypy")
+
+# Construct an instance of an aspect by applying configuration options to it.
+# rules_mypy doesn't provide a default instance which is a feature, since it
+# forces us to explicitly construct an instance and allows us to bypass the
+# defaults in the ruleset.
+mypy_aspect = mypy(
+    types = types,
+    mypy_cli = "@@//tools/mypy:mypy",
```

Comment:
`@@` is always a smell that the string is being evaluated into a Label in the wrong context (in the macro which lives in another repository)

Instead it's better to use a Label constructor to be explicit about the evaluation context being in this file.